wrapper/virt-v2v-wrapper.py

```diff
@@ -703,6 +703,11 @@ def handle_cleanup(self, data, state):
                         logging.info('Removing disk id=%s', disk_id)
                         disk_service.remove()
                         disk_ids.remove(disk_id)
+                    except self.sdk.NotFoundError:
+                        logging.info('Disk id=%s does not exist (already ' +
+                                     'removed?), skipping it',
+                                     disk_id)
+                        disk_ids.remove(disk_id)
```

Comment:
You are not permitted to remove an item from a list you are iterating over.  Even though this is pre-existing and mitigated by the fact that this happens in a while loop which will fix it up eventually, it'd be nice to get that fixed. It would also make it one second faster due to the `time.sleep(1)`.  You should change this:
```python
for disk_id in disk_ids:
```
to this:
```python
for disk_id in disk_ids[:]:
```

You can check it by yourself with this code:
```python
import time
a = list(range(10))
while len(a) > 0:
    for i in a:
        print(i)
        a.remove(i)
    time.sleep(1)
```

Reply:
As you say it's not a big deal because we're repeatedly going though the list. But it's a "nice to have", so yeah.

Reply:
Thanks to both - will fix these issues.